src/shared/tpm2-util.c

```diff
@@ -1838,6 +1847,81 @@ static int find_signature(
 #endif
 }
 
+int tpm2_calculate_name(const TPMT_PUBLIC *public, TPM2B_NAME *ret_name) {
```

Comment:
for the occasional reader a quick comment somewhere here explaining what a "name" in TPM2 parlance means. It's otherwise just too confusing... I don't think calling the hash of a public key a "name" is that common outside of tpm2 circles, no?

just a brief comment like `/* Calculates the "name" of a public key, i.e. the SHA256 digest of the TPM2 conformant marshalling of it */`

Reply:
updated with a explainer comment